src/desktop/apps/frieze_week/index.js

```diff
@@ -0,0 +1,65 @@
+import React from 'react'
+import { renderLayout as _renderLayout } from '@artsy/stitch'
+
+import adminOnly from '../../lib/admin_only'
+import JSONPage from '../../components/json_page/es6'
+import PageScaffold from 'desktop/components/fair_week_marketing/PageScaffold'
+import merge from 'lodash.merge'
+import queryString from 'query-string'
+
+let renderLayout = _renderLayout
+const SLUG = 'spring-art-fairs'
+const MARKETING_MODAL_ID = 'ca18'
```

Comment:
These are fine to be hard-coded?

Reply:
If there's a possibility those values will change, then I would recommend putting them in an ENV var